components/accelerator/nvidia/infiniband/component_read_class_test.go

```diff
@@ -1309,7 +1308,7 @@ func TestComponentReadClass_EventReporting(t *testing.T) {
 
 		// Should only include the known event type (drop), not the unknown one
 		summary := cr.Summary()
-		assert.Contains(t, summary, "mlx5_1 port 1 drop since")
+		assert.Contains(t, summary, "port drop mlx5_1")
```

Comment:
is it better that `port dropped of mlx5_1`?  or shall we make it easier to read or understand?  include all other messages in the component.